tests/draft2019-09/core/id.json

```diff
@@ -0,0 +1,138 @@
+[
+    {
+        "description": "Invalid use of fragments in location-independent $id",
+        "schema": {"$ref": "https://json-schema.org/draft/2019-09/schema"},
+        "tests": [
+            {
+                "description": "Identifier name",
+                "data": {
+                    "$ref": "#foo",
```

Comment:
We should probably also have at least one test that this is invalid even if the $ref points somewhere else or isn't present.

(I.e. `"$id": "$foo"`, but no `$ref` at all, or `"$ref": "someOtherValidRef"`)

Reply:
I added one test just below this one.